src/routes/Send/Send.tsx

```diff
@@ -209,60 +212,41 @@ const Send: FC = () => {
             ml="0"
           >
             <chakra.h4>I want to send</chakra.h4>
-            <InputGroup
-              width="auto"
-              fontSize="3rem"
-              alignItems="baseline"
-              my="1rem"
-              flexWrap="wrap"
-              justifyContent="center"
-            >
-              <NumberInput
-                value={amount}
-                step={0.00000001}
-                onChange={valueString => {
-                  try {
-                    const dotIndex = valueString.indexOf('.')
-                    if (
-                      dotIndex !== -1 &&
-                      valueString.substring(dotIndex + 1, valueString.length)
-                        .length > IRON_PRECISION
-                    ) {
-                      return
-                    }
-                    decodeIron(valueString)
-                    setAmount(valueString)
-                  } catch (err) {
+            <AmountInput
+              value={amount}
+              onChange={valueString => {
+                if (!valueString) {
+                  setAmount(null)
+                  return
+                }
+
+                if (valueString === '.') {
+                  setAmount('0.')
```

Comment:
If the user starts with an empty input and plops a `.`, we'll prepend a 0 for formatting purposes.